classifier_train.py

```diff
@@ -0,0 +1,87 @@
+import torch
+import torch.nn as nn
+from pdd.model import PDDModel
+from pdd.data_utils import unzip_data
+from pdd.data_utils import load_config
+from pdd.model import get_trained_model
+from train import prepare_datasets
+from train import fix_random_seed
+from train import split_on_train_and_test
+from pdd.model import Perceptron_classifier
+from torchbearer import Trial
+from torch.optim import Adam
+from pdd.trainer import forward_inputs_into_model
+from torchbearer.callbacks import Best
+
+
+def train_classifier(model, optimizer, criterion, metrics,
+                     train_em, train_labels, test_em, test_labels):
+    # optimizer = Adam(model.parameters())
+    checkpoint = Best(
+        'classifier.pt',
+        monitor='val_acc',
+        mode='max',
+        save_model_params_only=True)
+    trial = Trial(
+        model,
+        callbacks=[checkpoint],
+        optimizer=optimizer,
+        criterion=criterion,
+        metrics=['acc'])
+    trial.with_train_data(
+        torch.Tensor(train_em),
+        torch.Tensor(train_labels).long()).with_val_data(
+        torch.Tensor(test_em),
+        torch.Tensor(test_labels).long()).for_steps(100).run(40)
```

Comment:
Split different levels of functions for the better understanding:
```Python
trial.with_train_data(torch.Tensor(train_em),
                      torch.Tensor(train_labels).long())\
     .with_val_data(torch.Tensor(test_em),
                     torch.Tensor(test_labels).long())\
     .for_steps(100)\
     .run(40)
```

Or another way